enterprise/views.py

```diff
@@ -1642,6 +1645,7 @@ def post(self, request, enterprise_uuid, course_id):
         )
 
         # Create a link between the user and the enterprise customer if it does not already exist.
+
```

Comment:
[style nitpick] is there a reason for the spacing being added here?  Could we keep the comment above the code?

Reply:
@alex-sheehan-edx It's done